web/src/lib/components/Dashboard/Map/Map.svelte

```diff
@@ -132,9 +143,77 @@
 			);
 		}, 100);
 
```

Comment:
After you're done, could you please delete the circling point and the ottawa/toronto arc example I had? 
 :)    thanks